app/models/repository.rb

```diff
@@ -83,6 +83,12 @@ def self.search(
     end
   end
 
+  def self.assigned_in_project(project)
```

Comment:
shouldn't this be scope?

Reply:
and maybe more proper naming 'assigned_to_project

Reply:
they are not directly assigned to a project